README.md

```diff
@@ -1 +1,8 @@
-# engineering-posts
\ No newline at end of file
+# engineering-posts
+
+## How to add a blog post
+The easiest way to add a new blog post is forking this repository with your github user and write your blogpost in your fork of this repository. If you need help with how forking works, please refer to this guide by github: https://docs.github.com/en/get-started/quickstart/fork-a-repo.
```

Comment:
Maybe we could also document a short explanation why the approach via forking is needed vs. creating PRs directly on this repo.

Reply:
fair point 👍  Will do